BLSQ/iaso
# IA-2673: Orgunit change request dialog column size

While opening a change request rejected with only a reference instance on the right side of the dialog, the riht column is too big.

<img width="1618" alt="Screenshot 2024-02-02 at 10 57 38" src="https://github.com/BLSQ/iaso/assets/12494624/4ba25600-1314-43d9-bb56-2731858ecadc">

Related JIRA tickets : IA-2673
## Self proofreading checklist




## Changes

set a max width size for the columns

## How to test

Make sure you have a rejected change request  with reference instance change and no reference instance set on the org unit.
Open it in the dialog and make sure old and new values columns are the same


## Print screen / video

![Screenshot 2024-02-02 at 10 53 57](https://github.com/BLSQ/iaso/assets/12494624/2a6d71aa-557c-4e17-ae32-b1fc56e4ddea)
